proto/compiler.bzl

```
@@ -87,14 +87,24 @@ def go_proto_compile(go, compiler, protos, imports, importpath):
                 continue
             proto_paths[path] = src
 
-            out = go.declare_file(
-                go,
-                path = importpath + "/" + src.basename[:-len(".proto")],
-                ext = compiler.internal.suffix,
-            )
-            go_srcs.append(out)
+            if compiler.internal.suffixes:
+                for suffix in compiler.internal.suffixes:
+                    out = go.declare_file(
+                        go,
+                        path = importpath + "/" + src.basename[:-len(".proto")],
+                        ext = suffix,
+                    )
+                    go_srcs.append(out)
+            else:
+                out = go.declare_file(
+                    go,
+                    path = importpath + "/" + src.basename[:-len(".proto")],
+                    ext = compiler.internal.suffix,
+                )
+                go_srcs.append(out)
+
```

I would prefer unifying these code paths, e.g. like this (untested):

```suggestion
            suffixes = compiler.internal.suffixes
            if not suffixes:
                suffixes = [compiler.internal.suffix]
            for suffix in suffixes:
                out = go.declare_file(
                    go,
                    path = importpath + "/" + src.basename[:-len(".proto")],
                    ext = suffix,
                )
                go_srcs.append(out)

```

---

Addressed